funcs.php

```diff
@@ -107,6 +114,7 @@ function branches(
     $foundMinorBranchWithStableTag = [];
     foreach ($branches as $i => $branch) {
         // only remove minor branches, leave major branches in
+        // for developer-docs which has no tags, pretend that every branch has a tag
```

Comment:
Please move this comment to be directly above the conditional statement that actually does this.

Reply:
Updated